circle.yml

```diff
@@ -25,10 +25,12 @@ dependencies:
 
 test:
   override:
-    - cd "$PROJECT_PATH" && set -o pipefail && make test_integrations 2>&1 | tee test_integrations.log:
+   - case ${CIRCLE_NODE_INDEX} in 0) cd "$PROJECT_PATH" && make build_docker_test_image && make test_cover_ci;; 1) cd "$PROJECT_PATH" && make build_docker_test_image && make test_apps_ci;; 2) cd "$PROJECT_PATH" && make build_docker_test_image && make test_persistence_ci;; 3) cd "$PROJECT_PATH" && make build_docker_test_image && make test_p2p;; esac:
```

Comment:
ugly but works. `build_docker...` could be added as pre-run to the relevant make targets but that's gonna get re-worked with jenkins anyway